packages/terra-slider/src/Slider.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import styles from './Slider.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * When true, will disable the field.
+  */
+  isDisabled: PropTypes.bool,
+
+  /**
+   * Minimum value in the range.
+  */
+  minimumValue: PropTypes.number.isRequired,
+
+  /**
+   * Maximum value in the range.
+  */
+  maximumValue: PropTypes.number.isRequired,
+
+  /**
+  * Slider label value.
+  */
+  labelName: PropTypes.string.isRequired,
+
+  /**
+   * @private
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
+   */
+  intl: PropTypes.shape({ formatMessage: PropTypes.func }).isRequired,
+};
+
+const defaultProps = {
+  isDisabled: false,
+};
+
+/* eslint-disable-next-line react/prefer-stateless-function */
```

Comment:
This rule had to be disabled because you did not create a functional component.  However, it should not have been disabled.